docs/ref/models/fields.txt

```diff
@@ -319,6 +319,24 @@ characters that aren't allowed in Python variable names -- notably, the
 hyphen -- that's OK. Django quotes column and table names behind the
 scenes.
 
+
+``db_comment``
+--------------
+
+.. attribute:: Field.db_comment
```

Comment:
The documentation says that new features should have the `versionadded` block ([documentation](https://docs.djangoproject.com/en/3.2/internals/contributing/writing-documentation/#documenting-new-features)) like this:
```
.. versionadded:: 4.0
```
(It looks like the current in-development version is 4.0: [link](https://code.djangoproject.com/wiki/Version4.0Roadmap))
But I'm not 100% clear on whether the author should be adding that block or if that's the job of the release manager (in this case, Mariusz Felisiak). I'll post that question into the django-developers discussion group.

Reply:
Update: Carlton confirmed that adding the versionadded block is appreciated.